node/tests/fraud_test.go

```diff
@@ -0,0 +1,59 @@
+package tests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/fraud"
+	"github.com/celestiaorg/celestia-node/header"
+	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/celestia-node/node/tests/swamp"
+)
+
+/*
+Test-Case: Broadcast Fraud Proof to different nodes
```

Comment:
I don't quite understand who is broadcasting BEFP for this case. Could the naming be more detailed?

Reply:
Yes, thanks. Updated test case name.